R/uvhydrograph-data.R

```diff
@@ -241,7 +241,8 @@ getUvHydro <- function(ts, field, estimatedOnly = FALSE){
   
   if(!is.null(y) & !is.null(x)){
 
-    time <- as.POSIXct(strptime(x, "%FT%T"))
+    format <- "Ymd HMOS z"
+    time <- parse_date_time(x,format,tz = "America/Chicago",quiet=TRUE)
```

Comment:
Don't think we want to set any time zone, the timezone has to be based on the data, not the system running the parsing.